File: wallet/src/main/java/com/worldventures/wallet/analytics/firmware/action/UpdateInstallAction.java
package com.worldventures.wallet.analytics.firmware.action;


import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.janet.analytics.AnalyticsEvent;

@AnalyticsEvent(action = "wallet:settings:general:SmartCard Update:Ready Checklist:Install",
                trackers = AdobeTracker.TRACKER_KEY)
public class UpdateInstallAction extends FirmwareAnalyticsAction {
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/firmware/action/UpdateInstallAction.kt
package com.worldventures.wallet.analytics.firmware.action

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.AdobeTracker

@AnalyticsEvent(action = "wallet:settings:general:SmartCard Update:Ready Checklist:Install", trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class UpdateInstallAction : FirmwareAnalyticsAction()
